packages/datadog-plugin-jest/src/index.js

```diff
@@ -102,6 +98,28 @@ class JestPlugin extends CiPlugin {
       })
     })
 
+    this.addSub('ci:jest:worker-report:trace', data => {
+      const formattedTraces = JSON.parse(data).map(span => ({
+        ...span,
+        span_id: id(span.span_id),
```

Comment:
this `id` transformation is because interprocess communication only supports `json` (that's not quite true, but the 'advanced' serialization seems more complicated and its support more limited: https://nodejs.org/api/child_process.html#advanced-serialization), so we need to: 

* worker -> main process: from `Identifier` to JSON compatible.
* from JSON compatible to `Identifier` so that the old exporter can be used.